src/nvme/fabrics.c

```diff
@@ -412,9 +412,6 @@ static int build_options(nvme_host_t h, nvme_ctrl_t c, char **argstr)
 			errno = EINVAL;
 			return -1;
 		}
-		/* Use the default ctrl loss timeout if unset */
-                if (cfg->ctrl_loss_tmo == -1)
-			cfg->ctrl_loss_tmo = NVMF_DEF_CTRL_LOSS_TMO;
```

Comment:
Indeed, this needs to be removed; setting default values has been fixed with PR #110, and this is a left-over from previous attempts to set correct defaults.